src/main/resources/config/config.yaml

```diff
@@ -29,7 +29,8 @@ graphite:
 goCardless:
   # For initial integration we will use sandbox access token.
   # When in live mode: access token should be removed from the config and we should use partner integration instead.
-  accessToken: ${GDS_DIRECTDEBIT_CONNECTOR_GOCARDLESS_ACCESS_TOKEN:-}
+  accessToken: ${GDS_DIRECTDEBIT_CONNECTOR_GOCARDLESS_ACCESS_TOKEN:-change-me}
+  clientUrl: ${GDS_DIRECTDEBIT_CONNECTOR_GOCARDLESS_URL:-https://stubs.pymnt.localdomain/stub/gocardless}
```

Comment:
To consider for next PR, Given that this will go eventually to production environment I wouldn't default to a test value (I would set by default to a production URL and forcing the other envs to override it - this does not apply to secrets of course)

Reply:
yes, in the pr i am about to release I am removing the default for that, if it's not set, it will use the gocardless url for the account (TEST / LIVE)